test/test_jit.py

```diff
@@ -5620,6 +5620,120 @@ def backward_hook(module, grad_input, grad_output):
         with self.assertRaisesRegex(Exception, "backward hooks assigned"):
             torch.jit.trace(n, (torch.tensor(1.0),))
 
+    def test_scripting_hooks(self):
+        class Net(nn.Module):
+            def __init__(self):
```

Comment:
if the `__init__` is blank (i.e. only calls super's `__init__`) you can just omit it entirely